src/annotator/test/frame-observer-test.js

```diff
@@ -36,92 +28,98 @@ describe('annotator/frame-observer', () => {
     beforeEach(() => {
       container = document.createElement('div');
       document.body.appendChild(container);
+      sandbox.stub(console, 'warn');
+      $imports.$mock({
+        // Disable debouncing
+        'lodash.debounce': callback => callback,
+      });
+
```

Comment:
This seems like a straightforward approach to fixing this test issue, but made me curious if we have a convention around mocking third-party dependencies? I feel like we've discussed this before... @robertknight any thoughts?

Reply:
Mocking third-party dependencies is OK in some situations. The main risk to be aware of is that for tests where you do this, the test won't then catch situations where the dependency's API or behavior changes. TypeScript can still catch incompatible API changes in that situation. `lodash.debounce` has a very simple API so I think this is safe to do.

Reply:
`lodash.debounce` is mocked in several places in our code.